mods/d2k/rules/aircraft.yaml

```diff
@@ -82,6 +82,7 @@ frigate:
 		Repulsable: False
 		MaximumPitch: 20
 		CruiseAltitude: 2048
+		LandableTerrainTypes: Rock, Concrete
```

Comment:
IMO it would be best to leave this to #16574 or #16509.

Reply:
Ok, will remove it.